### PR TITLE
refactor (BackupFinish): refactor code

### LINKED ISSUE #358: backup-finish
[Epic 356](https://github.com/haqq-network/haqq-wallet/issues/356)

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #358

## Proposed Changes

  - Changed the imports to absolute:
  -Changed textAlign to center props
  -Moved text to i18n
  -Change the colors from the constants to similar ones in Color
  -Transferring the screen logic to components

### CODE REVIEW

File: src/components/backUpFinishPopUp.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React, {useEffect} from 'react';
```
Comment (vivalaakam) [inline src/components/backUpFinishPopUp.tsx:1]: rename file please in kebab-case

File: src/components/backUpFinishPopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  Spacer,
+  Text,
+} from '@app/components/ui';
+import { useTypedNavigation } from '@app/hooks';
+import { I18N } from '@app/i18n';
+
+const animationSize = Dimensions.get('window').width - 116;
+
+export const BackUpFinishPopUp = () => {
+  const navigation = useTypedNavigation();
```
Comment (vivalaakam) [inline src/components/backUpFinishPopUp.tsx:20]: move navigation to screens and call throw callback

File: src/components/backUpFinishPopUp.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          style={{width: animationSize, height: animationSize}}
+        />
+      </Spacer>
+      <Text t4 center i18n={I18N.backupFinishcongrat}/>
+      <Text t4 center i18n={I18N.backupFinishSuccess} style={page.title}/>
+      <Button
+        style={page.button}
+        variant={ButtonVariant.contained}
+        i18n={I18N.backupFinishFinish}
+        onPress={() => {
```
Comment (vivalaakam) [inline src/components/backUpFinishPopUp.tsx:42]: call callback from screen component

File: src/components/backUpFinishPopUp.tsx (hunk 1 above)
Comment (vivalaakam) [reply src/components/backUpFinishPopUp.tsx:1]: and backup and popup should be written without a capital `Up`

File: src/components/backUpFinishPopUp.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  }, []);
+
+  return (
+    <PopupContainer style={page.popupContainer}>
+      <Spacer style={page.container}>
+        <LottieWrap
+          source={require('../../assets/animations/backup-success-animation.json')}
+          autoPlay
+          loop={false}
+          style={{width: animationSize, height: animationSize}}
```
Comment (vivalaakam) [inline src/components/backUpFinishPopUp.tsx:33]: can be moved in styles

File: src/components/home-settings/home-settings.tsx [hunk 5]
```diff
@@ -14,6 +14,7 @@ export const HomeSettings = () => {
   const {theme} = useUser();
```
Comment (vivalaakam) [inline src/components/home-settings/home-settings.tsx:14]: rollback, please, cause makes no changes

File: src/i18n.ts [hunk 6]
```diff
@@ -165,6 +165,10 @@ export enum I18N {
   welcomeCreateWallet,
   welcomeLedgerWallet,
   welcomeRestoreWallet,
+  backupFinishcongrat,
```
Comment (vivalaakam) [inline src/i18n.ts:168]:
```suggestion
  backupFinishCongratulation,
```

**Comment by vivalaakam** [comment]
please make sure you have the precommit hook running and that it works correctly

**Review by vivalaakam — CHANGES_REQUESTED** [review]
please make sure you have the precommit hook running and that it works correctly

File: src/i18n.ts [hunk 7]
```diff
@@ -366,4 +370,7 @@ const en: Record<I18N, string> = {
   [I18N.welcomeCreateWallet]: 'Create Wallet',
   [I18N.welcomeLedgerWallet]: 'Connect',
   [I18N.welcomeRestoreWallet]: 'I already have a wallet',
+  [I18N.backupFinishcongrat]: 'Congratulations!',
+  [I18N.backupFinishSuccess]: "You've successfully protected your wallet.!",
```
Comment (vivalaakam) [inline src/i18n.ts:374]:
```suggestion
  [I18N.backupFinishSuccess]: "You've successfully protected your wallet.",
```

File: src/components/backUpFinishPopUp.tsx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+        i18n={I18N.backupFinishFinish}
+        onPress={() => {
+          navigation.getParent()?.goBack();
+        }}
+      />
+    </PopupContainer>
+  );
+};
+
+const page = StyleSheet.create({
```
Comment (vivalaakam) [inline src/components/backUpFinishPopUp.tsx:50]: use `createTheme` instead `StyleSheet.create`

File: src/i18n.ts (hunk 7 above)
Comment (samandarkuchkarov) [reply src/i18n.ts:374]: ready

File: src/components/backUpFinishPopUp.tsx (hunk 3 above)
Comment (samandarkuchkarov) [reply src/components/backUpFinishPopUp.tsx]: Should I do with all functions in this app or when using  navigation hook

**Pull request closed without merging**